src/crt/slland.src

```diff
@@ -0,0 +1,68 @@
+; Performs 48 bit ("short long long") bitwise and
+; 
+; Arguments:
+; uiy = Most significant bytes of arg 1
+; ubc = Least significant bytes of arg 1
+; ude = Most signficant bytes of arg 2
+; uhl = Least significant bytes of arg 2
+; 
+; Returns:
+; ude:uhl = ude:uhl & uiy:ubc
+
+	assume	adl=1
+
+	section	.text
+	public	__slland
+__slland:
+	push af
+
+	; Set up stack  stuff for later
+
+	push ix
```

Comment:
I recommend using IY as a frame pointer after you push it later on, then you don't have to save and restore IX